docs/underscore.array.builders.js.md

```diff
@@ -208,8 +222,8 @@ _.keepIndexed([1,2,3], function(k) {
 If you return either `null` or `undefined` then the result is dropped from the resulting array:
 
 ```javascript
-_.keepIndexed(['a','b','c'], function(k, v) { 
-  if (k === 1) return v; 
+_.keepIndexed(['a','b','c'], function(k, v) {
```

Comment:
Hi @jgonggrijp sorry, I didn't see the PR very deeply, but these look like to me only whitespace specific changes and also the one at line 294 below, is it done due to some eslint formatting work or if not deliberate,then maybe better to remove them as it might cause confusion.

Reply:
You are right about that. I put them on separate commits, though. So you can [review the commits one by one](https://github.com/documentcloud/underscore-contrib/pull/240/commits/d6f6ddad7b2d0d4ccc1fb1873689323c794d8620) in order to skip most of the whitespace changes (click "Prev"/"Next" in the top right to move between the commits).